MozziGuts.h

```diff
@@ -76,18 +76,18 @@ x...11.......Arduino Mega  *broken since Jan 2015 \n
 ....14........Teensy  \n
 x..B5........Teensy2  \n
 x..B5(25)..Teensy2++  \n
-x..A14.....Teensy3.1  \n
+x..A14.....Teensy 3.0, 3.1 and 3.2  \n
 ....13	.......Sanguino  \n
 
-On Teensy 3.1 STANDARD and STANDARD_PLUS are the same, providing 16384Hz sample rate and 12 bit resolution on pin A14/ADC.
-The Teensy 3.1 DAC output does not rely on PWM.
+On Teensy 3.* STANDARD and STANDARD_PLUS are the same, providing 16384Hz sample rate and 12 bit resolution on pin A14/ADC.
+The Teensy 3.* DAC output does not rely on PWM.
 */
 
 
 /** @ingroup core
 Used to set AUDIO_MODE to HIFI.
 
-HIFI (not for Teensy 3.1)
+HIFI for AVR (not for Teensy 3.* and ARM)
```

Comment:
Ha! Just found another small reason to nag ;-)

STM32 actually has support for HIFI mode (and is an ARM). ESP8266 does not, and is not an ARM. Same a few lines below.

Reply:
oh whoops. This is gonna get complicated. You are doing PWM and HIFI on STM32 but most of the STM32 boards I have seen have a pair of 12-bit DAC's. It's gonna be a conditional compilation zoo in the near future.

Reply:
Incidentally, most of the libraries are getting this wrong because Arduino only just sorted out some conventions about board #defines (and Teensy is not "on board" yet). Teensy code conditions on the particular chip for each Teensy board rather than a board description. So far this works because Paul is the only one employing those chips. Using an architecture description like ARM or STM32 or SAMD may be too broad. My favorite way you have addressed this so far is your list of tested boards... I wish I had time to test all the Mozzi examples on all the different boards I have (30+).

Reply:
> oh whoops. This is gonna get complicated. You are doing PWM and HIFI on STM32 but most of 
> the STM32 boards I have seen have a pair of 12-bit DAC's. It's gonna be a conditional compilation 
> zoo in the near future.

Yes, all but the cheap STM32's have built-in DAC, and one day we'll want to support those, natively. Add to that I plan on adding PDM output (more CPU intensive, but potentially much higher quality) for STM32, too, and all those \#ifdefs are pretty messy, already. That why I insisted on you adding those comments! I really hope we can think up a sane way of organizing this zoo, one day...